lib/rules/valid-lang.json

```diff
@@ -1,7 +1,7 @@
 {
   "id": "valid-lang",
-  "selector": "[lang], [xml\\:lang]",
-  "matches": "not-html-matches",
+  "selector": "[lang]:not(html), [xml\\:lang]:not(html)",
+  "matches": "has-lang-text-matches",
```

Comment:
Could we switch these around? I.e. only look for the text if the lang value is invalid?